frontend/public/components/node.tsx

```diff
@@ -243,7 +247,7 @@ const Details = ({obj: node}) => {
           </thead>
           <tbody>
             {_.map(images, (image, i) => <tr key={i}>
-              <td>{image.names.find(name => !name.includes('@')) || image.names[0]}</td>
+              <td className="co-break-word">{image.names.find(name => !name.includes('@')) || image.names[0]}</td>
```

Comment:
I suggest `co-break-all` here since it's a long identifier. Better to let it break at cleanly at the edge of the table cell